src/gam.py

```diff
@@ -7103,25 +7111,19 @@ def doCreateVaultMatter():
       body[u'description'] = sys.argv[i+1]
       i += 2
     elif myarg in [u'collaborator', u'collaborators']:
-      collaborators = sys.argv[i+1].split(u',')
-      cd = buildGAPIObject(u'directory')
-      for collaborator in collaborators:
-        collaborator_id = convertEmailAddressToUID(collaborator, cd)
-        if not collaborator_id:
-          print u'ERROR: failed to get a UID for %s. Please make sure this is a real user.' % collaborator
-          sys.exit(4)
-        collaborators_map.append({u'email': collaborator, u'id': collaborator_id})
+      if not cd:
+        cd = buildGAPIObject(u'directory')
+      validateCollaborators(sys.argv[i+1], collaborators, cd)
```

Comment:
It seems confusing to pass collaborators in here. Just make collaborators the return value of validateCollaborators.